File: Assignments/DataBases/app/src/main/java/com/example/neera/databases/models/Todo.java
package com.example.neera.databases.models;

/**
 * Created by Neera on 28/09/17.
 */

public class Todo {
    int id;
    String task;
    boolean done;

    public Todo(int id, String task, boolean done) {
        this.id = id;
        this.task = task;
        this.done = done;
    }

    public Todo(String task, boolean done) {
        this.task = task;
        this.done = done;
    }

    public int getId() {
        return id;
    }

    public String getTask() {
        return task;
    }

    public boolean isDone() {
        return done;
    }
}

File: Lecture17/FirebaseDB/app/src/main/java/com/example/neera/firebasedb/models/Todo.kt
package com.example.neera.firebasedb.models

/**
 * Created by Neera on 22/10/17.
 */
class Todo {
    var task: String = ""
    var isDone: Boolean = false
    var uid: String? = null

    constructor() {
    }

    constructor(task: String, done: Boolean) {
        this.task = task
        this.isDone = done
    }
}